### ISSUE TITLE
Latin SSA support revision

### ISSUE THREAD

**Issue by vv-monsalve** [TypeTogether/Playpen-Sans#15]
Hi @josescaglione, I'm reviewing the glyph coverage based on our Glyphsets as we discussed today.
I'll be reporting here the ones listed as "missing"

#### ARABIC

<img width="519" alt="PPS-Arabic-support" src="https://github.com/user-attachments/assets/5a35e52b-50e2-433e-8e16-b67229b06a22">

**Comment by josescaglione** [comment]
yes, this is correct, all three Arabic glyphs will be in the merged fonts

**Comment by vv-monsalve** [comment]
> yes, this is correct, all three Arabic glyphs will be in the merged fonts

Excellent! I'm adding here the report from Shaperglot for Arabic using the command `shaperglot check /path/to/font ar_Arab` 
Please use [Shaperglot](https://github.com/googlefonts/shaperglot) to ensure the support of all the intended languages: Arabic, Kurdish, Dari, Dogri, Persian, Qashqa'i, Talysh, and Urdu

````
Font does not fully support language 'ar_Arab'
 * FAIL: Shaper didn't attach alefabove-ar to None
 * FAIL: Shaper didn't attach madda-ar to None
 * FAIL: Shaper didn't attach fathatan-ar to None
 * FAIL: Shaper didn't attach dammatan-ar to None
 * FAIL: Shaper didn't attach fatha-ar to None
 * FAIL: Shaper didn't attach damma-ar to None
 * FAIL: Shaper didn't attach shadda-ar to None
 * FAIL: Shaper didn't attach sukun-ar to None
 * FAIL: .medi version of ARABIC LETTER ALEF MAKSURA; both buffers returned space=1+0|alefMaksura-ar=1+768|space=0+0
 * FAIL: .init version of ARABIC LETTER ALEF MAKSURA; both buffers returned space=0+0|alefMaksura-ar=0+768
 ````

**Comment by vv-monsalve** [comment]
After pulling the latest files at commit `c5d152b` 47 glyphs are missing to meet the [Latin African glyphset](https://github.com/googlefonts/glyphsets/blob/main/data/results/txt/nice-names/GF_Latin_African.txt)

<img width="510" alt="Screenshot 2024-09-17 at 15 59 31" src="https://github.com/user-attachments/assets/cbc95eb6-9df3-479a-babe-e4c4d681a783">

**Comment by josescaglione** [comment]
> After pulling the latest files at commit `c5d152b` 47 glyphs are missing to meet the [Latin African glyphset](https://github.com/googlefonts/glyphsets/blob/main/data/results/txt/nice-names/GF_Latin_African.txt)
> 

Hi @vv-monsalve , this is our report about the missing codepoints. This is part of the documentation we processed and provided in February. 

Note that the ramshorn is missing and that we cannot find any instances of doublemacronbelow. Do you know what languages use it?

<img width="1071" alt="Screen Shot 2024-09-18 at 09 26 00" src="https://github.com/user-attachments/assets/dd401449-f86d-41b0-97f2-24c8b96c4096">

**Comment by vv-monsalve** [comment]
The GF_Ltin_African was reviewed and redefined this year by @moyogo. Denis, could you please review the above list and provide feedback on the items marked in red?

**Comment by moyogo** [comment]
Thanks for looking into this. It’s good someone double checks this stuff, given the size of the data, errors is not impossible. A lot of the data comes from various sources but I’ve tried to verify as much as possible.

The latest GF_Latin_African is generated from [gflanguages](http://googlefonts/gflanguages) data.

Given the references that were verified and listed in the gflanguage data files, most of the red characters either are requirements for current orthographies or are auxiliary (and appear in some current reference documents).

Looking at the imports from before references were added, Jukun Takum needs references but the red charaters overlap with Tangale for which there are references. I’ll add references for Jola-Kasa, Tunisian Darija and Tangale that I verified now. Tangale requirements needs to be updated.


| UC | LC | Language (use) | Note
| -- | -- | -- | --
| Ȧ  | ȧ  | Timne (auxiliary)<br>Harari (auxiliary) |
| Ḇ  | ḇ  | Masana (auxiliary) |
| Ƃ  |    | Liberian Dan-Gio | Alternate to Ɓ Bhook and should be available as Bhook glyph alternate as historical form of Bhook still used in Liberia Dan-Gio. Some works use Ƃ character instead
| Ḋ  | ḋ  | Harari (auxiliary) |
| Ꟈ  | ꟈ  | Moro | Alternate to đ for some authors, should be both separate characters and glyph alternates to /Dstroke/dstroke
| Ḕ  | ḕ  | Tunisian Darija (auxiliary) |
| Ƹ  | ƹ  | Riffian (auxiliary) | Alternate to ɛ for some authors, separate characters.
| Ḣ  | ḣ  | Harari (auxiliary) |
| Ḳ  | ḳ  | Jukun Takum<br>Tangale |
| Ḵ  | ḵ  | Uduk |
| Ṃ  | ṃ  | Tunisian Darija (auxiliary) |
| Ṇ  | ṇ  | Baka, DRC/South Sudan<br>Ngas |
| Ᵽ  | ᵽ  | Bandial |
| Ṛ  | ṛ  | Baka, DRC/South Sudan<br>Morokodo<br>Ndogo<br>Kabyle<br>Riffian<br>Tachelhit<br>Central Atlas Tamazight<br>Tunisian Darija (auxiliary) |
| Ȓ  | ȓ  | Riffian (auxiliary) |
| Ṡ  | ṡ  | Harari (auxiliary) |
| Ṫ  | ṫ  | Harari (auxiliary) |
| Ṿ  | ṿ  | Baka, DRC/South Sudan |
| Ṽ  | ṽ  | Mundang |
| Ȳ  | ȳ  | Mango (auxiliary)<br>Sar |
|    | ˀ  | Mendankwe-Nkwen (auxiliary)<br>Southern Giziga | Alternate to ʔ or ɂ in some works in Mendankwe-Nkwen.
|    | ˈ  | Lobi (auxiliary)<br>Lehar (auxiliary)<br>Ndut (auxiliary) | Alternate to ' 0027, ’ 2019 or ʼ 02BC in some works, or alternate to ꞌ A78C before it was added to Unicode.
|    | ◌̑  | Riffian (auxiliary) | see Ȓ ȓ
|    | ◌̥  | Jola-Kasa | 
|    | ◌͟  | Sudanese Arabic<br>Uduk | Some Uduk references use ṯẖ instead of t͟h, but macron below shouldn’t connect so ṯẖ is problematic.

Let me know if you need more information.

**Comment by t-grace** [comment]
@moyogo thank you for the explanations. Regarding /doublemacronbelowcomb, are t and h the only characters involved?

**Comment by moyogo** [comment]
@t-grace /doublemacronbelowcomb is only used in Uduk with th as far as the collected language data shows. There are several orthographies that haven't been verified and added so it's possible it is used in a few others.

Generally speaking, the character may be used in other pairs in a few transliteration systems, for example in some UN transliteration of Arabic or academic ones like the Encyclopedia of Islam: t͟h, d͟j, k͟h, d͟h, s͟h, g͟h, z͟h, k͟h, t͟s, s͟h, s̱h︭č̱ (not sure), y͟u, y͟a. There's also been some use in dictionaries but this is even more specific. Unicode does provide a character for trigraphs or longer sequences: U+FE2D, which should connect with macrons below of the letters before and after as a single macron. These are out of scope as far as I can tell.

As a side note, in some practical orthographies underlined letters are used, a legacy use from typewriter or text formatting. These could be represented with U+0332 combining low line but using U+0331 combining macron below, or characters composed with it, is more appropriate. U+0332 is currently out of scope and not in any language data collected.

**Comment by t-grace** [comment]
This is very helpful for development. Thank you!

**Comment by vv-monsalve** [comment]
I've turned this issue into a dedicated one for Latin African. I'll file another one for the other scripts.

**Comment by vv-monsalve** [comment]
@josescaglione the following are the reported missing glyphs for the SSA: `icapitalsmall` and `hmacronbelow`.
However, before working on them, let's wait to receive the first feedback for the SSA.

**Comment by vv-monsalve** [comment]
The LGC SSA font is considered ready. Closing this issue here.
